packages/twenty-front/src/modules/users/graphql/queries/getCurrentUserAndViews.ts

```diff
@@ -0,0 +1,103 @@
+// This query cannot be put in the graphQL folder because it cannot be generated by the graphQL codegen.
+import { gql } from '@apollo/client';
+
+export const GET_CURRENT_USER_AND_VIEWS = gql`
+  query GetCurrentUserAndViews {
+    currentUser {
+      id
+      firstName
+      lastName
+      email
+      canImpersonate
+      supportUserHash
+      workspaceMember {
+        id
+        name {
+          firstName
+          lastName
+        }
+        colorScheme
+        avatarUrl
+        locale
+      }
+      defaultWorkspace {
+        id
+        displayName
+        logo
+        domainName
+        inviteHash
+        allowImpersonation
+        subscriptionStatus
+        featureFlags {
+          id
+          key
+          value
+          workspaceId
+        }
+      }
+    }
+    views {
```

Comment:
later we should generate this from our mapMetadataToQuery hooks but this would require to have the system standard object available in the FE no matter if we have fetched /metadata already or not. We could also leverage fragment for this part